framework/auth/core.py

```diff
@@ -977,6 +977,22 @@ def get_activity_points(self, db=None):
         db = db or framework.mongo.database
         return analytics.get_total_activity_count(self._primary_key, db=db)
 
+    def disable_user(self):
```

Comment:
Minor: `user.disable_user()` is redundant, since this is an instance method; this can just be `disable()` or `disable_account()`.